src/main/java/com/ericsson/gerrit/plugins/eiffel/GerritModule.java

```diff
@@ -45,11 +48,18 @@ protected void configure() {
                                      .to(MessageQueueHandler.class);
 
         // Example of how to register plugin configuration to the project screen
-        // bind(ProjectConfigEntry.class).annotatedWith(
-        // Exports.named(EiffelPluginConfiguration.ENABLED))
-        // .toInstance(new ProjectConfigEntry("Enable Eiffel messaging",
-        // false));
-
+        bind(ProjectConfigEntry.class).annotatedWith(Exports.named(EiffelPluginConfiguration.ENABLED))
+                .toInstance(new ProjectConfigEntry("Enable Eiffel messaging", false));
+        bind(ProjectConfigEntry.class).annotatedWith(Exports.named(EiffelPluginConfiguration.FILTER))
+                .toInstance(new ProjectConfigEntry("Filter branch", ""));
+        bind(ProjectConfigEntry.class).annotatedWith(Exports.named(EiffelPluginConfiguration.FLOW_CONTEXT))
+                .toInstance(new ProjectConfigEntry("Flow Context", ""));
+        bind(ProjectConfigEntry.class).annotatedWith(Exports.named(EiffelPluginConfiguration.REMREM_PUBLISH_URL))
+                .toInstance(new ProjectConfigEntry("REMReM Publish URL", ""));
+        bind(ProjectConfigEntry.class).annotatedWith(Exports.named(EiffelPluginConfiguration.REMREM_USERNAME))
+                .toInstance(new ProjectConfigEntry("REMReM Username", ""));
+        bind(ProjectConfigEntry.class).annotatedWith(Exports.named(EiffelPluginConfiguration.REMREM_PASSWORD))
```

Comment:
So the password is stored in plain text right now? And is this field visible in the GUI? If we go for this in the first iteration, I think you should create an issue immediately about changing it later.

Reply:
Yes, it is plain text and there is no type that can say this field as a password. 

I added a comment on top of the field so that we need to modify it later.